lib/jobs/install-os.js

```diff
@@ -65,7 +65,6 @@ function installOsJobFactory(
      */
     InstallOsJob.prototype._validateOptions = function() {
         assert.string(this.context.target);
-        assert.string(this.options.completionUri);
```

Comment:
Hmm, I'll check, it's possible I need to rebase this out, but I do believe it does not exist in the task definition anymore.